db/fixtures/miq_event_definitions.csv

```diff
@@ -223,6 +223,12 @@ containergroup_scheduled,Pod Scheduled,Default,container_operations
 containergroup_compliance_check,Pod Compliance Check,Default,compliance
 containergroup_compliance_passed,Pod Compliance Passed,Default,compliance
 containergroup_compliance_failed,Pod Compliance Failed,Default,compliance
+containergroup_containercreated,Pod Container Created,Default,container_operations
+containergroup_containerfailed,Pod Container Failed,Default,container_operations
+containergroup_containerkilling,Pod Container Killing,Default,container_operations
+containergroup_containerstarted,Pod Container Started,Default,container_operations
+containergroup_containerstopped,Pod Container Stopped,Default,container_operations
+containergroup_containerunhealthy,Pod Container Unhealthy,Default,container_operations
```

Comment:
:heart: the names with "Pod" at the beginning, this is presently the only way for user to know which events belong in which policies